### DISCUSSION TITLE
‘aving a play

### DISCUSSION THREAD

**Discussion by MunzyTdub** [keiwando/toonsquid#11] [Ideas]
Just got the app a few days ago. Really love it. Havent tested out everything yet but really looking forward to learning more about the limitations. Love the brush feel and the simple timeline and it runs very smooth even with videos imported. I’d love to see palm rejection and split screen as well.

Overall its pretty amazing! Big ups to the developer. 🔥

https://user-images.githubusercontent.com/115199445/194448905-f021e8bd-64ba-4e94-a2b0-b449f96a24f1.MOV

**Comment by trinhpxl** [comment]
hello fellow toonsquid user !!! i often check these forums as a user LOL; anywho, im loving this app as well :D i would love palm rejection and split screen, too ! ^_^

**Comment by keiwando** [comment]
Thanks for sharing the video and for your kind words! I'm glad that you have been enjoying ToonSquid so far :)

I have now created a new thread to collect information about palm rejection issues. If you have any more details, please report them [here](https://github.com/keiwando/toonsquid/discussions/13).

**Reply by MunzyTdub** [reply]
No worries. Thanks for the hard work! I’ll try get a screen recording of the issues I have in future, especially with palm rejection.
